crypto/bn/bn_prime.c

```diff
@@ -279,6 +279,7 @@ static int probable_prime(BIGNUM *rnd, int bits, prime_t *mods)
     char is_single_word = bits <= BN_BITS2;
 
  again:
+    /* XXX: Not all primes are private */
     if (!BN_priv_rand(rnd, bits, BN_RAND_TOP_TWO, BN_RAND_BOTTOM_ODD))
```

Comment:
The comment `Not all primes are private` right in front of a call to `BN_priv_rand()` is a little confusing. And by the way: What does `XXX:` mean?

Reply:
Maybe I should replace it with TODO